### ISSUE TITLE
data-duration attribute does not work

### ISSUE THREAD

**Issue by olaulau** [nadchif/html-duration-picker.js#180]
`data-duration` attribute doesn't work, whereas `value` attribute works

tested on firefox & chromium, same behavior

https://codepen.io/olaulau/pen/XWLjxZr

even the `src/index.html` does not work as expected .

**Comment by dKally** [comment]
Yes, I've also observed this behavior with the `data-duration` attribute. I've submitted a pull request with a fix.
